python/ray/data/tests/test_actor_pool_map_operator.py

```diff
@@ -126,6 +128,37 @@ def test_pending_to_running(self):
         assert pool.num_idle_actors() == 0
         assert pool.num_free_slots() == 3
 
+    def test_restarting_to_running(self):
```

Comment:
```suggestion
    def test_restarting_to_alive(self):
```
Let's use the term "alive", because "running" actors include both alive and restarting actors